pvlib/irradiance.py

```diff
@@ -882,12 +887,12 @@ def reindl(surface_tilt, surface_azimuth, dhi, dni, ghi, dni_extra,
     Parameters
     ----------
     surface_tilt : numeric
-        Surface tilt angles in decimal degrees. The tilt angle is
+        Surface tilt angles. [°] The tilt angle is
         defined as degrees from horizontal (e.g. surface facing up = 0,
         surface facing horizon = 90)
 
     surface_azimuth : numeric
-        Surface azimuth angles in decimal degrees. The azimuth
+        Surface azimuth angles. [°] The azimuth
         convention is defined as degrees east of north (e.g. North = 0,
         South=180 East = 90, West = 270).
```

Comment:
What about in this case @kandersolar? Since there is more significant amount of text after the unit, not just a "see :term: ..." that could go into the brackets.
Does the following (not) look more normal?
```suggestion
        Surface azimuth angles [°]. The azimuth
        convention is defined as degrees east of north (e.g. North = 0,
        South=180 East = 90, West = 270).
```

Reply:
If the unit is to go in the middle of the paragraph, I agree your suggestion is the better of those two options.  But I don't see why it should go in the middle like that.  I'd still favor putting it at the end, like this:
```suggestion
        Surface azimuth angles. The azimuth
        convention is defined as degrees east of north (e.g. North = 0,
        South=180 East = 90, West = 270). [°]
```

In this specific case, I'd also ditch the 0/180/90/270 sentence and replace it with a `see :term:...` anyway.

Reply:
Makes sense 👍

> In this specific case, I'd also ditch the 0/180/90/270 sentence and replace it with a `see :term:...` anyway.

I am reminded of the discussion in #1421, in particular [this comment ](https://github.com/pvlib/pvlib-python/issues/1421#issuecomment-1076860942)

Reply:
See #2448